new-lamassu-admin/src/components/inputs/Select.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useSelect } from 'downshift'
+import _ from 'lodash/fp'
+import classnames from 'classnames'
+
+import { ReactComponent as Arrowdown } from '../../styling/icons/action/arrow/regular.svg'
+
+import styles from './Select.styles'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles(styles)
+
+function Select ({ items, ...props }) {
+  const classes = useStyles()
+
+  const {
+    isOpen,
+    selectedItem,
+    getToggleButtonProps,
+    getLabelProps,
+    getMenuProps,
+    getItemProps
+  } = useSelect({
+    items,
+    selectedItem: props.selectedItem,
+    onSelectedItemChange: item => {
+      props.onSelectedItemChange(item.selectedItem)
+    }
+  })
+
+  const selectClassNames = {
+    [classes.select]: true,
+    [classes.selectFiltered]: selectedItem !== props.default
+  }
+
+  return (
+    <div className={classnames(selectClassNames)}>
+      <label {...getLabelProps()}>Level</label>
```

Comment:
We are. I just realized it should probably be passed into the component, though, instead of a fixed value.

![image](https://user-images.githubusercontent.com/5789328/67568765-fef71600-f724-11e9-9ebc-d94fdbd9e22f.png)